lib/core/errors.c

```diff
@@ -5,6 +5,9 @@ int __log_level = 3;
 int __enable_syslog = 1;
 const int __max_log_msg_len = 1024;
 
+enum ReturnCode __status_code = CLEAN_EXIT;
+char __err_msg[1024] = "Hello there";
```

Comment:
This message will get printed unless the other change is also applied.
```suggestion
char __err_msg[__max_log_msg_len] = "not set";
```